packages/destination-actions/src/destinations/adobe-target/index.ts

```diff
@@ -17,8 +17,24 @@ const destination: DestinationDefinition<Settings> = {
           'Your Adobe Target client code. To find your client code in Adobe Target, navigate to **Administration > Implementation**. The client code is shown at the top under Account Details.',
         type: 'string',
         required: true
+      },
+      bearer_token: {
+        label: 'Bearer Token',
```

Comment:
@marinhero Can we please rename this to "Authentication Token"?

```suggestion
        label: 'Authentication Token',
```